go.mod

```diff
@@ -3,27 +3,19 @@ module github.com/zcash-hackworks/lightwalletd
 go 1.12
 
 require (
+	github.com/adityapk00/lightwalletd v0.0.0-20191111191511-e881e44e962c
```

Comment:
We shouldn't need this right? It'd add Aditya's fork of lightwalletd as a dependency to our own.